microsetta_private_api/client/myfoodrepo.py

```diff
@@ -0,0 +1,195 @@
+
+        return self._json_to_model(r.content)
+
+    def cohorts(self):
+        """Obtain cohorts associated with the study
+
+        Returns
+        -------
+        object
+            SimpleNamespace translation of
+
+            {
+              "data": {
+                "study": { "codename": "healthy_humans" },
+                "cohorts": [
+                  { "codename": "programmers" },
+                  { "codename": "doctors" },
+                  { "codename": "pilots" }
+                ]
+              }
+            }
+
+        Notes
+        -----
+        Return body from: https://myfoodrepo.org/api_doc/v1/studies_and_cohorts
+        """
+        return self._req('GET', f'/studies/{self.study_codename}/cohorts')
+
+    def create_subj(self, cohort):
+        """Create a new subject within a cohort
+
+        Returns
+        -------
+        object
+            SimpleNamespace translation of
+
+            {
+              "data": {
+                "subject": {
+                  "key": "abcdef",
+                  "created_at": "2021-03-01T14:30:00.123456Z",
+                  "first_attached_at": null,
+                  "expired_at": null
+                }
+              }
+            }
+
+        Notes
+        -----
+        Return body from: https://myfoodrepo.org/api_doc/v1/studies_and_cohorts
+        """
+        return self._req('POST', f'/studies/{self.study_codename}'
+                                 f'/cohorts/{cohort}/subjects')
+
+    def read_subj(self, cohort, subj):
+        """Reads details about a subject
+
+        Returns
+        -------
+        object
+            SimpleNamespace translation of
+
+            {
+              "data": {
+                "subject": {
+                  "key": "abcdef",
+                  "created_at": "2021-03-01T14:30:00.123456Z",
+                  "first_attached_at": null,
+                  "expired_at": null
+                }
+              }
+            }
+
+        Notes
+        -----
+        Return body from: https://myfoodrepo.org/api_doc/v1/studies_and_cohorts
+        """
+        return self._req('GET', f'/studies/{self.study_codename}'
+                                f'/cohorts/{cohort}'
+                                f'/subjects/{subj}')
+
+    def update_subj(self, cohort, subj, expired_at):
+        """Update the expired_at attribute of a subject
+
+        Returns
+        -------
+        object
+            SimpleNamespace translation of
+
+
+        Notes
+        -----
+        Return body from: https://myfoodrepo.org/api_doc/v1/studies_and_cohorts
+        """
+        return self._req('PUT', f'/studies/{self.study_codename}'
+                                f'/cohorts/{cohort}'
+                                f'/subjects/{subj}',
+                         json={'subject': {'expired_at': str(expired_at)}})
+
+    def delete_subj(self, cohort, subj):
+        """Remove a subjust being tracked
```

Comment:
```suggestion
        """Remove a subject being tracked
```

...feedback from EPFL team